src/pj_observation.c

```diff
@@ -112,3 +168,25 @@ int pj_show_triplet (FILE *stream, const char *banner, PJ_TRIPLET point) {
         i += fprintf(stream, "\n");
     return i;
 }
+
+PJ *pj_pj (PJ_CONTEXT *ctx, char *definition) {
+    if (0==ctx)
+        return pj_init_plus_ctx (pj_get_default_ctx(), definition);
+    return pj_init_plus_ctx (ctx, definition);
+}
+
+PJ *pj_pj_argv (PJ_CONTEXT *ctx, int argc, char **argv) {
+    return pj_init_ctx (ctx, argc, argv);
+}
+
+PJ_CONTEXT *pj_ctx (PJ *P) {
```

Comment:
You can do the same with the API in ```proj_api.h``` as documentend here: http://proj4.org/development/threads.html
This, and the one below, is just to get consistent naming across all ```pj_ctx*```functions. I am not really able to comment on why you would need that functionality though...